openedx/features/discounts/tests/test_applicability.py

```diff
@@ -171,6 +171,6 @@ def test_holdback_expiry(self):
         with patch('openedx.features.discounts.applicability.stable_bucketing_hash_group', return_value=0):
             with patch(
                 'openedx.features.discounts.applicability.datetime',
-                Mock(now=Mock(return_value=datetime(2020, 8, 1, 0, 1, tzinfo=pytz.UTC)), wraps=datetime),
+                Mock(now=Mock(return_value=datetime(2021, 8, 1, 0, 1, tzinfo=pytz.UTC)), wraps=datetime),
```

Comment:
It won't work after 2021. One way is to get the current year and add 1 year into it.

Reply:
fixed